src/components/screen-renderer.vue

```diff
@@ -1,56 +1,86 @@
+            />
           </g>
         </g>
       </svg>
-      {{ $t('Loading...') }}
+      {{ $t("Loading...") }}
     </div>
-    <component ref="component" :is="component" :vdata="value" :_parent="_parent" :_initial-page="currentPage"
+    <component
+      :is="component"
+      ref="component"
+      :vdata="value"
+      :_parent="_parent"
+      :_initial-page="currentPage"
       @submit="submit"
       @asyncWatcherTriggered="onAsyncWatcherOn"
       @asyncWatcherCompleted="onAsyncWatcherOff"
     />
-    <screen-renderer-error v-if="showErrors && building.error" v-model="building" />
-    <watchers-synchronous ref="watchersSynchronous"/>
+    <screen-renderer-error
+      v-if="showErrors && building.error"
+      v-model="building"
+    />
+    <watchers-synchronous ref="watchersSynchronous" />
   </b-container>
 </template>
 
 <script>
-import Json2Vue from '../mixins/Json2Vue';
-import CurrentPageProperty from '../mixins/CurrentPageProperty';
-import WatchersSynchronous from '@/components/watchers-synchronous';
-import ScreenRendererError from '../components/renderer/screen-renderer-error';
-import { cloneDeep, isEqual } from 'lodash';
+import { cloneDeep, isEmpty, isEqual } from "lodash";
+import Json2Vue from "../mixins/Json2Vue";
+import CurrentPageProperty from "../mixins/CurrentPageProperty";
+import WatchersSynchronous from "@/components/watchers-synchronous";
+import ScreenRendererError from "../components/renderer/screen-renderer-error";
 
 export default {
-  name: 'screen-renderer',
+  name: "ScreenRenderer",
   components: { WatchersSynchronous, ScreenRendererError },
-  mixins: [ Json2Vue, CurrentPageProperty ],
+  mixins: [Json2Vue, CurrentPageProperty],
   data() {
     return {
       currentDefinition: null,
-      codigo: '',
-      self: this,
```

Comment:
removed `self` since 1. It wasn't being used and 2. goes agaisn't some javascript rules such as 
```var this = that`